app/frontend/components/pages/login/loginPage.tsx

```diff
@@ -160,9 +165,11 @@ class LoginPage extends Component<Props, {isDropdownOpen: boolean}> {
         {authMethod === 'file' && <KeyFileAuth />}
       </div>
     )
+    getErrorBannerContent()
```

Comment:
I think this belongs rather to componentDidMount callback than the rendering logic and for this usecase and also since the function mutates the state, IMHO the verb "load" would fit more. "get" kind of leads the reader into thinking it's not a mutating operation which is not true

Reply:
true, I was considering calling the function in componentDidMount, but then I thought about a situation when the function would call some request to backend health check API, (this could take some time) while the banner would not be rendered, and then suddenly after the function returns something the whole view would jump down to render the banner. This way the type of banner is known before render. Either way, we are either postponing the render by the time the function returns or postponing the render of the banner while the function returns. It makes sense to me either way but lets call it in `componentDidMount` then.